src/content/keyAutoAdd/testWallet.ts

```diff
@@ -166,6 +167,25 @@ async function revokeKey(accountId: string, walletId: string, keyId: string) {
     throw new Error(`Failed to revoke key: ${await res.text()}`);
   }
 }
+
+function normalizeWalletAddress(urlOrPaymentPointer: string) {
+  const url = toWalletAddressUrl(urlOrPaymentPointer);
+  if (IS_INTERLEDGER_CARDS && url.startsWith('https://ilp.dev')) {
```

Comment:
```suggestion
  if (IS_INTERLEDGER_CARDS && url.startsWith('https://ilp.dev/')) {
```

To fix above CodeQL alert?